app/commands/decidim/participations/admin/update_participation.rb

```diff
@@ -73,6 +77,26 @@ def participation_limit_reached?
           end
         end
 
+        def recipient_role_will_change?
```

Comment:
Could you make the method name more explicit or add a comment to explain what is its goal please, it's not really clear

Reply:
@ludivinecp it checks if the role of the recipient will change, I am open to other names